src/robot.ts

```diff
@@ -160,7 +169,26 @@ class Robot {
   }
 }
 
-module.exports = (...args) => new Robot(...args)
+export const createRobot = (options: RobotOptions) => new Robot(options)
+
+export interface EventWithEventField {
+  event: string
+}
+
+// The TypeScript definition for cache-manager does not export the Cache interface so we recreate it here
```

Comment:
It looks like there is an interface for this: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/cache-manager/index.d.ts

Reply:
There is, but it's not exported. We could get out of sync if they change their function signature, but also that should just create a compiler error. Thoughts on leaving it for now?